docs/README.md

```diff
@@ -488,17 +496,14 @@ boosted-blog
 There are two things to do when creating a read model:
 
 1. Define who is authorized to query or subscribe it
-1. and filter out unneeded fields from the entity
-
-Read models and commands compound the public API of a Booster application. With the
-`CreatePost` command we authorized `all` to execute it, and this time we'll do the same
-for the `PostReadModel`.
+1. Add the logic of the projection functions, where you can filter, combine, etc., the entities fields.
```

Comment:
👍

Reply:
This should be:

```suggestion
2. Add the logic of the projection functions, where you can filter, combine, etc., the entities fields.
```

Reply:
You don't have to do this. Markdown will render it properly so that you don't have to keep the numbers up to date ;-)